libs/application/templates/children-residence-change/src/fields/components/index.ts

```diff
@@ -0,0 +1 @@
+export { default as DescriptionText } from './DescriptionText/DescriptionText'
```

Comment:
Should this be a tsx file since we are importing and exporting a tsx component?

Reply:
Good catch ⚾ 🧤